src/peerctl/settings/__init__.py

```diff
@@ -143,6 +144,8 @@
 
 settings_manager.set_option("AUTOPEER_ENABLED", False)
 
+settings_manager.set_list("CORS_ALLOWED_ORIGINS", [], envvar_element_type=str)
```

Comment:
@jason20c this one i can also see getting a spot in https://github.com/fullctl/fullctl/blob/main/src/fullctl/django/settings/__init__.py